### PR TITLE
Make kibana start after elastic search.

### PR DESCRIPTION
**Pull request by john-gress**

### CODE REVIEW

**Comment by yannick-polius** [comment]
:+1:

File: init/kibana.conf [hunk 1]
```diff
@@ -1,4 +1,15 @@
+# Kibana 4
+#
+description     "Kibana 4"
+
+start on started elasticsearch
+stop on (stopped probelogger or runlevel [!345])
```
Comment (KjellKod) [inline init/kibana.conf]: as with the other PRs I'm not sure about the shutdown order. Why not shutdown once ProbeManager is stopped?

**Comment by craig-cogdill** [comment]
:+1:

**Comment by yannick-polius** [comment]
:+1:

**Comment by john-gress** [comment]
merge problems. Closing.

**Pull request closed without merging**